### PR TITLE
Fedotova 54 add lesson

### PR DESCRIPTION
**Pull request by Winchestery**
Added HTML/CSS layout of Add Lesson
Mock-up
image

Layout
image

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -5,14 +5,16 @@ import { Provider } from 'react-redux';
...
 import { Counter } from './features';
 import Icon from './icon.js';
+import {AddLesson} from "./features/add-lesson";
 
 const store = configureStore();
 
 export const App = () => (
     <Provider store={store}>
-      <h1>Hello, I'm App Component!</h1>
+      <h1 pt={5}>Hello, I'm App Component!</h1>
```
Comment (sarhan-azizov) [inline src/app.js:13]: !

File: src/features/add-lesson/add-lesson.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import styles from './add-lesson.module.scss';
+
+
+export const AddLesson = () => {
+    
+    const names = [
```
Comment (sarhan-azizov) [inline src/features/add-lesson/add-lesson.js:7]: https://www.npmjs.com/package/faker

File: src/features/add-lesson/add-lesson.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        "Taras Tarasov",
+        "Taras Tarasov",
+        "Taras Tarasov",
+        "Taras Tarasov",
+        "Taras Tarasov",
+        "Taras Tarasov",
+    ];
+    
+    return (
+        <div className={`container  ${styles.page}`}>
```
Comment (sarhan-azizov) [inline src/features/add-lesson/add-lesson.js:23]:
```suggestion
        <div className={ctx(styles.page, 'container')}>
```

File: src/features/add-lesson/add-lesson.js [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+                                <th scope="col" className='text-center'>Mark</th>
+                                <th scope="col" className='text-center'>Presence</th>
+                            </tr>
+                            </thead>
+                            <tbody>
+                            { names.map((value, index) => {
+                                return <tr>
+                                    <th scope="row">{ index+1 }</th>
+                                    <td><a href="#">{ value }</a></td>
+                                    <td><input className={`${styles.align}`} type="number" max='12' min='2'/></td>
```
Comment (sarhan-azizov) [inline src/features/add-lesson/add-lesson.js:69]: className={`${styles.align}`}

className={styles.align}


align ?

**Review by PavelKundenko — APPROVED** [review]

**Review by sarhan-azizov — APPROVED** [review]

**Review by Anastasiia0999 — APPROVED** [review]

File: src/features/add-lesson/add-lesson.js (hunk 2 above)
Comment (Winchestery) [reply src/features/add-lesson/add-lesson.js:7]: done

File: src/app.js (hunk 1 above)
Comment (Winchestery) [reply src/app.js:13]: deleted

File: src/features/add-lesson/add-lesson.js (hunk 3 above)
Comment (Winchestery) [reply src/features/add-lesson/add-lesson.js:23]: changed

File: src/features/add-lesson/add-lesson.js (hunk 4 above)
Comment (Winchestery) [reply src/features/add-lesson/add-lesson.js:69]: changed

**Pull request merged**